includes/class-sensei-db-query-learners.php

```diff
@@ -105,6 +99,41 @@ private function build_query() {
 		return $sql;
 	}
 
+	/**
+	 * Get last activity date by users.
+	 *
+	 * @param int[] $user_ids User IDs to get the last activity date.
+	 *
+	 * @return array Last activity date array.
+	 */
+	private function get_last_activity_date_by_users( $user_ids ) {
+		global $wpdb;
+
+		$in_placeholders = implode( ', ', array_fill( 0, count( $user_ids ), '%s' ) );
+
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.NoCaching, WordPress.DB.DirectDatabaseQuery.DirectQuery
+		$results = $wpdb->get_results(
+			// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared, WordPress.DB.PreparedSQLPlaceholders.UnfinishedPrepare -- Placeholders created dinamically.
```

Comment:
Just writing to confirm that you are using `phpcs:disable` on purpose. As opposed to `phpcs:ignore` this will disable the check until the end of the current file not just on the following line.

Reply:
Good catch! It wasn't on purpose. Fixed in: https://github.com/Automattic/sensei/pull/5104/commits/4e602d0ff36fc2eb24a110852c0b4b3b7bbcafc5